minebooks2.py

```diff
@@ -0,0 +1,205 @@
+from loadBooks import *
+import string
+from vaderSentiment.vaderSentiment import SentimentIntensityAnalyzer
+import nltk
+from collections import defaultdict #frequwords
+import random
+import operator
+
+
+def removegutnburg(text):
+	"""
+	Removes the Gutenberg license information for so that the text can 
+	be analyzed.
+
+	"""
+	
+	licence = "ject Gutenberg Association / Carnegie Mellon University" #last line in license
+	bookstart = "by William Shakespeare" # set to Act V to
+	bookend = "THE END"
+	start_pos = text.find(licence) + 60 				#finds the end of the licensing agreement	
+	start_reading = text.find(bookstart,start_pos) + 22 #all of the books start with "title" by William Shakespeare
+	end_reading = text.find(bookend,start_reading ) 	#finds THE END at the end of the book
+	return (text[start_reading:end_reading])			#returns text of the play
+
+
+def loadjustbooks(playfiles):
+	"""
+	Takes a list of play names and a list of text files of those plays.
+	Removes Gutenberg license, newlines and punctuation from the play text.
+
+	Ex. Makes a list of all of the text of the comedic plays
+
+	"""
+
+	plays = []
+
+	
+	for i in playfiles:
+		play = opensavedbook(i)		#opens play text file
+		just_play = removegutnburg(play) #removes gutenberge license
+		just_play = RemoveAllButLettersAndSpaces(just_play) #removes newlines and punctuation
+		plays.append(just_play)		#adds current play to the play list
+
+	return plays #returns list of plays
+
+def RemoveAllButLettersAndSpaces(mystring):
+	"""
+	removes special characters and punctuation from play texts.
+	>>>  RemoveAllButLettersAndSpaces("\r\nqwetr.,")
+	'qwe2tr'
+	"""
+	mystring = mystring.replace('\r', ' ')
+	mystring = mystring.replace('\n', ' ')
+	mystring = mystring.replace('\'', ' ')
+	mystring = mystring.replace('[', '')
+	mystring = mystring.replace(']', '')
+	mystring = mystring.replace('.', '')
+	mystring = mystring.replace('?', '')
+	mystring = mystring.replace('!', '')
+	mystring = mystring.replace(';', '')
+	mystring = mystring.replace(':', '')
+	mystring = mystring.replace('-', '')
+	mystring = mystring.replace('\"', '')
+	mystring = mystring.replace(',', '')
+	return mystring.lower() #makes everything lower case words like "The" and "the" are the same

```

Comment:
There's a cleaner way to do this with a for loop. Make a list of all the characters you want to eliminate and iterate through that list rather than writing down every single character as a distinct line.